### ISSUE TITLE
~ Dk Star Zone ~ Report

### ISSUE THREAD

**Issue by al3xc1985** [cmangos/wotlk-db#399]
1. If u cancel the eye of archerus control, when u get back to your body, your speed increase so much like 4 / 5x. 

~ More coming ~

**Comment by MantisLord** [comment]
> 1. If u cancel the eye of archerus control, when u get back to your body, your speed increase so much like 4 / 5x.  The speed bonus is permanent, not even .modify speed 1 or logout/login does not modify  the speed. Also even if you get down to open world, your speed will be so high

It's a permanent speed aura that gets stuck on player when you cancel the Eye of Acherus.

51923 Eye of Acherus Flight (Boost) must be removed if Eye of Acherus vehicle/buff is removed
52637 See Acherus Invisibility too?

This can be done easily with spell script if anyone is inclined.

**Comment by al3xc1985** [comment]
As I see we have no testers for wotlk, I will try to fit the job. As for scripting, I don;t know how to do it
@Grz3s hope to see you back in here again. What do you say? are you recoming back?
@MantisLord  Ty for the fast reply,

**Comment by Grz3s** [comment]
@xfyrry and I .. spent lots of time on polishing DK zone:
- that aura was an issue before - and we did fix it... (came back like bumerang after trillion @killerwife updates)
- Citizen of Heavenshire - need to recheck that -- very easy to fix ...
-3 and 4 Quest Death's Challange - if im correct this was made in sd2 by @xfurry 100years ago.. it may need relook ... 
My pc still lays in box somewhere... (+ im quite busy with other stuff - so this must wait) or if other good person wants to look - we all will be happy ;)

**Comment by killerwife** [comment]
I am looking at wotlk specific issues but I am staying away from content until ur back :) I want to stabilize the wotlk core across the board so its not a boomerang like u said.

**Comment by al3xc1985** [comment]
@Grz3s & @killerwife  I have some exams this period, but I shall try my best to help Grz3s in his work just by testing, and give a complete report upon this dk zone and some other zones, as I can

**Comment by xfurry** [comment]
Oh boy...
I guess spell auras are probably broken again.
I will try to test it soon, but can't promise anything.

**Comment by al3xc1985** [comment]
@killerwife have u done something with the speed aura in recent changes? cause after I finished quest with eye of archerus, I canceled it, but this time from the player spell bar,, and my movement speed is back to normal. 

Is it because last time I canceled the eye from his pet bar?

**Comment by al3xc1985** [comment]
7. Quest The Gift That Keeps on Giving does not work anymore at all ... works now... dunno how...but it works

**Comment by al3xc1985** [comment]
> @xfurry and I .. spent lots of time on polishing DK zone:
> 
> * that aura was an issue before - and we did fix it... (came back like bumerang after trillion @killerwife updates)
> * Citizen of Heavenshire - need to recheck that -- very easy to fix ...
> * 3 and 4 Quest Death's Challange - if im correct this was made in sd2 by @xfurry 100years ago.. it may need relook ...
>   My pc still lays in box somewhere... (+ im quite busy with other stuff - so this must wait) or if other good person wants to look - we all will be happy ;)

I was rebuild the core again...and testing

**Comment by al3xc1985** [comment]
I need to tell u guys, that I even reextracted mmap/vmaps/etc to recheck , not just compiling and pushing again...it might have been something messy in here... I will retest an other time, also, the quest with the eye and cancel it in both ways to see what is going weird and where

**Comment by al3xc1985** [comment]
@xfurry @Grz3s @killerwife this is the final result after several tests on the dk zone. Honestly it is working very very nice and not so many bugs left at all

**Comment by al3xc1985** [comment]
next stop, each class spells and talents

**Comment by al3xc1985** [comment]
first post updated a little bit

**Comment by xfurry** [comment]
> first post updated a little bit

Are there still issues with "the light of dawn"? I thought I already fixed those last year.
Maybe I can get some info from the retail Wotlk classic and compare our version with the original one.

**Comment by al3xc1985** [comment]
Yes,  u can see the things I saw not good in cmangos battles  reported

**Comment by al3xc1985** [comment]
I mean @xfurry check my report list about it

**Comment by Grz3s** [comment]
`Quests like this can be taken during warchief blessing? I belieave that this are neutral quests and should NOT be available to u, until u complete the warchief blessing. OR am I wrong?`
i just checkd it on blizz ... and its exact the same...
![dk](https://user-images.githubusercontent.com/2374069/210641362-a1260d09-a818-477a-84a3-1a6fbad7ed5e.jpg)
